sdk/src/main/java/com/atlan/model/assets/Column.java

```diff
@@ -911,6 +911,11 @@ public ISQL getParent() {
                         .tableQualifiedName(tableQualifiedName)
                         .tablePartition(TablePartition.refByQualifiedName(parentQualifiedName));
                 break;
+            case SnowflakeDynamicTable.TYPE_NAME:
+                builder.tableName(parentName)
+                        .tableQualifiedName(parentQualifiedName)
+                        .snowflakeDynamicTable(SnowflakeDynamicTable.refByQualifiedName(parentQualifiedName));
+                break;
```

Comment:
This needs to be added to the Freemarker template that's used to generate these asset classes, too, or the next time we regenerate this will just be overwritten.

Reply:
Sorry I forgot this was generated.